python/src/aiconfig/editor/server/server.py

```diff
@@ -114,17 +117,83 @@ def create() -> FlaskResponse:
     state.aiconfig = AIConfigRuntime.create()  # type: ignore
     return HttpResponseWithAIConfig(message="Created new AIConfig", aiconfig=state.aiconfig).to_flask_format()
 
+@app.route('/api/test_streaming', methods=["POST"])
+def test_streaming():
+    EXCLUDE_OPTIONS = {
+        "prompt_index": True,
+        "file_path": True,
+        "callback_manager": True,
+    }
+    state = get_server_state(app)
+    request_json = request.get_json()
+    num_stream_steps : int = request_json.get("num_stream_steps", 10)
+    print(f"{num_stream_steps=}")
+    print(f"{type(num_stream_steps)=}")
+    def generate(num_stream_steps: int):
+        prompt : Prompt = state.aiconfig.get_prompt('get_activities')
+        for i in range(num_stream_steps):
+            time.sleep(1)
```

Comment:
why are we sleeping?

Reply:
Just to prove that the streaming is going as planned and I can see outputs bit-by-bit